src/components/Summary/AlterationSummary.vue

```diff
@@ -232,6 +238,23 @@ export default class AlterationSummary extends Mixins(
   /** Prop to perform validation. */
   @Prop() readonly validate: boolean
 
+  /** The future effective fee for Alteration. */
+  futureEffectiveFees: number = 0
+
+  async mounted () {
+    // fetches the fee prices to display in the text
+    const feePrices = await this.fetchFeePrices()
+    this.futureEffectiveFees = feePrices.futureEffectiveFees
```

Comment:
if `feePrices` request fails `feePrices.futureEffectiveFees` could return a type error. You might want to optional chain here `feePrices?.futureEffectiveFees` or set a fallback value to prevent `futureEffectiveFees` being defined as `null` or `undefined` with could bubble up into the UI.

Reply:
Right.
You possibly throw an error in `fetchFeePrices()` so you need to catch it here or `mounted` will fail and possibly the whole component won't render.
And it's possibly that `feePrices` doesn't contain the property `futureEffectiveFees` so you need to check it and have a reasonable fallback.

Reply:
Yes, good point indeed. But then I should catch it and emit a "fetchError" for the parent component deal with it (Alteration page in this case)?

Reply:
That's a good question. I had to think about this...

1. We don't handle fee errors in any other case (eg, in Fee Summary), so I think we don't need to think too much about it here.

2. If we fail to get the fees, I don't think we should block this alteration filing altogether. I think we can probably display some reasonable fallback text instead. Ask Scott/Tracey about this, or come up with something yourself and pass it by us. This should be a rare scenario so (1) don't crash, and (2) try to show something reasonable, even if it's only "unknown" for the fee numbers.

Reply:
ok. I changed to what Tracey mentioned in the ticket - if we don't get the data, the text will be displayed without the values but without breaking the text.